src/config/2022.json

```diff
@@ -866,7 +865,8 @@
         "authors"
       ],
       "avatar_url": "17236402",
-      "github": "imeugenia"
+      "github": "imeugenia",
+      "twitter": "jevgeniazi"
```

Comment:
@imeugenia I added your Twitter profile here but feel free to update to remove if you'd prefer not to have it. you can also add a website and LinkedIn profile if you so desire.